shanoir-ng-datasets/src/main/java/org/shanoir/ng/examination/service/ExaminationServiceImpl.java

```diff
@@ -67,7 +68,8 @@
 	
 	@Override
 	public void deleteById(final Long id) throws EntityNotFoundException {
-		Examination exam = examinationRepository.findOne(id);
+		Examination exam = examinationRepository.findById(id).orElse(null);
```

Comment:
As this will provoke directly a NPE further, maybe we should directly throw something we control here ? (But it's quite complicated to do this everywhere...)

Reply:
changed, and now throws ENFE